mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxViewModelStore.kt

```diff
@@ -103,9 +103,12 @@ class MvRxViewModelStore(private val viewModelStore: ViewModelStore) {
             } else {
                 ownerArgs
             }
-            map[it] = (host as? Fragment)
-                    ?.let { fragment -> restoreViewModel(fragment, viewModelsState.getParcelable(it), arguments) }
-                    ?: restoreViewModel(host as FragmentActivity, viewModelsState.getParcelable(it), arguments)
+            val holder = viewModelsState.getParcelable<MvRxPersistedViewModelHolder>(it)!!
+            map[it] = when(host) {
```

Comment:
nit: missing a space before after `when`

Reply:
Are you sure about this?